src/OpenTelemetry/Trace/BatchSpanExportProcessorOptions.cs

```diff
@@ -0,0 +1,91 @@
+// <copyright file="BatchSpanExportProcessorOptions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Diagnostics;
+using System.Security;
+using OpenTelemetry.Internal;
+
+namespace OpenTelemetry.Trace
+{
+    public class BatchSpanExportProcessorOptions : BatchExportProcessorOptions<Activity>
```

Comment:
I think that `BatchExportActivityProcessorOptions` would be a little "more English". I have also seen `SpanProcessor` term in the [specs](https://github.com/open-telemetry/opentelemetry-specification). WDYT?

Reply:
`BatchExportActivityProcessorOptions` sounds good to me.

Reply:
Addressed 7dfb9354

I just noticed that we have `BatchExportActivityProcessor` which lives in the `OpenTelemetry` namespace. Does it mean that `BatchExportActivityProcessorOptions` should be placed in the same namespace (instead of `OpenTelemetry.Trace`)?

Reply:
> I just noticed that we have `BatchExportActivityProcessor` which lives in the `OpenTelemetry` namespace.

IIRC it was a mistake and when folks realized that it's already too late to correct it (unless we deprecate it or introduce some breaking changes).

> Does it mean that `BatchExportActivityProcessorOptions` should be placed in the same namespace (instead of `OpenTelemetry.Trace`)?

I vote for `OpenTelemetry.Trace` namespace, and I saw that you already did this 👍

Reply:
> IIRC it was a mistake and when folks realized that it's already too late to correct it (unless we deprecate it or introduce some breaking changes).

Yes this is true. We can keep this in OpenTelemetry.Trace ns.